scripts/module/teleporter.nut

```diff
@@ -13,24 +13,22 @@ teleporters <- {
   "y": Vector( 557.677, -886.057, 432.612 ) };
 
 function teleport(player, location) {
-  local location = location.tolower();
-  local pos = false;
-  
+  location = location.tolower();
   if (location in teleporters) {
-    pos = teleporters[location];
+    TeleportPlayer(player, teleporters[location], 0);
+    return true;
   }
-  if (pos == false) {
-    return false;
-  }
-  TeleportPlayer(player, pos, 0);
-  return true;
+  return false;
 }
 
 
 function onPlayerCommand( player, cmd, text ) {
   cmd = cmd.tolower();
   if(cmd == "t") {
-    if (!teleport(player, text)) {
+    if(!text || typeof text != "string"){
+      MessagePlayer("Usage: /" + cmd + " <string>", player);
+    }
+    else if (!teleport(player, text)) {
```

Comment:
This fusion of `else` and `if` is not wise.

The teleporter logic is like a separate function, so it should be isolated like it (even if it's not really a function, but just part of the scope):

```
if (...) {
  stuff
} else {
  this scope could also be a function
}
```

`else if` is better for switch-case like constructs